RNA-seq/nb_cell_line_DESeq2.Rmd

```diff
@@ -0,0 +1,209 @@
+---
+title: "Neuroblastoma Cell Line: gene-level summarization with `tximport`"
+output:   
+  html_notebook: 
+    toc: true
+    toc_float: true
+---
+
+**CCDL 2018**
+
+In this notebook, we'll perform an analysis to identify the genes that are 
+differentially expressed in _MYCN_ amplified vs. nonamplified neuroblastoma 
+cell lines. 
+
+These RNA-seq data are from 
+[Harenza, et al. _Scientific Data._ 2017.](https://doi.org/10.1038/sdata.2017.33).
+
+More information about DESeq2 can be found in the [excellent vignette](https://bioconductor.org/packages/release/bioc/vignettes/DESeq2/inst/doc/DESeq2.html) 
+from Love, Anders, and Huber from which this is adapted 
+(see also: [Love, Anders, and Huber. _Genome Biology_. 2014.](https://doi.org/10.1186/s13059-014-0550-8))
+
+DESeq2 takes unnormalized counts or estimated counts does the following:
+
+* Estimates size factors
```

Comment:
Do you want to provide hyperlinks for these - i'm guessing a subset of participants might want more information